screeps-game-api/src/objects/macros.rs

```diff
@@ -86,3 +86,33 @@ macro_rules! impl_room_object_properties {
         )*
     );
 }
+
+
+// Macro for mass implementing `StructureProperties`, `PartialEq` and `Eq` for a type. 
```

Comment:
I think this could be a doc comment with `//!`, but it's not a big deal - I can add in the exclamation marks afterwards.

Reply:
I changed the `//` for `///`, but I'll wait until #19 is merged before moving this one with the other.